core/src/main/java/org/apache/accumulo/core/client/security/tokens/KerberosToken.java

```diff
@@ -59,6 +59,20 @@ public KerberosToken(String principal) throws IOException {
   }
 
   /**
+   * Creates a Kerberos token for the specified principal using the provided keytab. The principal and keytab combination are verified by attempting a log in.
+   *
```

Comment:
This blank line is meaningless without a <p> or <br> in between, and may be subject to reformatting during a build.